ts/src/bitfinex2.ts

```diff
@@ -747,13 +727,18 @@ export default class bitfinex2 extends Exchange {
                 const currencyId = this.safeString (this.safeValue (pair, 1, []), 0);
                 if (currencyId === cleanId) {
                     const network = rawType ?? this.safeNetwork (networkId);
+                    const networkStatus = this.safeValue (indexed['networkStatus'], networkId, []);
+                    const depositEnable = this.safeInteger (networkStatus, 1, 0) === 1;
+                    const withdrawEnable = this.safeInteger (networkStatus, 2, 0) === 1;
```

Comment:
Goddamn... this is way too much for me 😆

Reply:
Yep, the response type of bitfinex is terrible af. We take the element on index 1 and if it's equal to 1 -> the deposit is activated. By default, we put it to 0